tests/rocprof-sys-rocm-tests.cmake

```
@@ -84,6 +84,23 @@ rocprofiler_systems_add_test(
     REWRITE_FAIL_REGEX "0 instrumented loops in procedure transpose")
 
 if(ROCPROFSYS_USE_ROCM)
+    set(NAVI_REGEX "gfx(10|11|12)[A-Za-z0-9][A-Za-z0-9]")
```

```suggestion
    set(NAVI_REGEX "gfx(10|11|12)[A-Fa-f0-9][A-Fa-f0-9]")
```
Only need to check hexadecimal